packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -120,8 +122,21 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
     );
   }
 
-  private _getIcon(): JSX.Element {
-    return <i className={ css('ms-Icon', this.props.icon || 'ms-Icon--FavoriteStarFill') } />;
+  private _getIcon(starSize?: number): JSX.Element {
+    if (starSize && starSize < 1) {
+      return (
+        <span className={ css('ms-rating-halfStarContainer', styles.halfStarContainer, 'ms-Icon', this.props.icon || 'ms-Icon--FavoriteStarFill') } >
+          <i className={ css('ms-Rating-star', styles.star,
+            'is-selected ' + styles.starIsSelected,
+            { ['is-disabled ' + styles.starIsDisabled]: this.props.disabled },
```

Comment:
Because half star is compound of two stars - background grey star and foreground truncated star. Background star is always grey so we need to again format the foreground star.